hedera-node/cli-clients/src/main/java/com/hedera/services/cli/signedstate/DumpScheduledTransactionsSubcommand.java

```diff
@@ -0,0 +1,252 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.cli.signedstate;
+
+import static com.hedera.services.cli.signedstate.DumpTokensSubcommand.getListFormatter;
+import static com.hedera.services.cli.signedstate.DumpTokensSubcommand.getOptionalFormatter;
+import static com.hedera.services.cli.utils.ThingsToStrings.quoteForCsv;
+import static java.util.Objects.requireNonNull;
+
+import com.hedera.node.app.service.mono.legacy.core.jproto.JKey;
+import com.hedera.node.app.service.mono.state.adapters.MerkleMapLike;
+import com.hedera.node.app.service.mono.state.submerkle.EntityId;
+import com.hedera.node.app.service.mono.state.submerkle.RichInstant;
+import com.hedera.node.app.service.mono.state.virtual.EntityNumVirtualKey;
+import com.hedera.node.app.service.mono.state.virtual.schedule.ScheduleVirtualValue;
+import com.hedera.services.cli.signedstate.DumpTokensSubcommand.FieldBuilder;
```

Comment:
If this class is used outside DumpTokensSubcommand isn't it better to have it as a separate class?